src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationLoaderSecurity7.java

```diff
@@ -84,6 +87,14 @@
         log.debug("Index is: {}", securityIndex);
     }
 
+    /**
+     * Checks if audit config doc is present in security index
+     * @return true/false
+     */
+    boolean isAuditConfigDocPresentInIndex() {
+        return isAuditConfigDocPresentInIndex.get();
```

Comment:
How is it enforced that `isAuditConfigDocPresentInIndex` is initialized before `isAuditConfigDocPresentInIndex()` is called?

Reply:
As part of bootstrapping the configs are reloaded https://github.com/sujithvm/security/blob/auditlog/src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationRepository.java#L192

This would ensure whether doc is present in the index or not.